apps/worker/src/app/workflow/usecases/message-matcher/message-matcher.usecase.ts

```diff
@@ -101,7 +101,8 @@ export class MessageMatcher extends Filter {
         if (singleRule) {
           const result = await this.processFilter(variables, children[0], command, filterProcessingDetails);
           if (!prefiltering) {
-            await this.createExecutionDetails.execute(
+            await this.executionLogQueueService.add(
```

Comment:
I believe we should consider having the`createdAt` timestamp generated here, instead of being generated by mongoose at write time. So the user can better see the timed history of an execution detail. 

Wdyt?

Reply:
i think thats a good suggestion, eventhough it take milliseconds to write in most of the scenarios, in some cases queue might be busy and the write might be delayed. I will make that change